pyspark/requirements.txt

```diff
@@ -0,0 +1,6 @@
+dask-geopandas>=0.1.0a4
+odc-algo>=0.2.0a3
+odc-stac>=0.2.0a6
+
+# a temporary workaround to load the latest snapshot
+https://geotrellis-test.s3.amazonaws.com/raster-frames/pyrasterframes-0.10.1.dev0-py3-none-any.whl
```

Comment:
I hope that is fine for now, we'd need a fixed release, could be done as a follow up PR though.

Reply:
Note: this is a special RasterFrames build with the GDAL 3.4.x support.